### PR TITLE
alpine: Update to 2.21.999 and update URLS

### PR DESCRIPTION
**Pull request by neheb**
alpine seems to have a new home. Only a git repository now. No release tarballs.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @pesintta 
Compile-tested: mvebu

### CODE REVIEW

File: mail/alpine/Makefile [hunk 1]
```diff
@@ -8,17 +8,21 @@
...
-PKG_VERSION:=2.20
-PKG_RELEASE:=2
+PKG_VERSION:=2.21.999
+PKG_RELEASE:=1
 
-PKG_SOURCE_URL:=http://patches.freeiz.com/alpine/release/src/
+PKG_SOURCE_PROTO:=git
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_HASH:=ed639b6e5bb97e6b0645c85262ca6a784316195d461ce8d8411999bf80449227
+PKG_SOURCE_URL:=git://repo.or.cz/alpine.git
```
Comment (diizzyy) [inline mail/alpine/Makefile]: @neheb 
Please change this to http instead (https has broken cert)

**Comment by neheb** [comment]
This is using git, not HTTPS.

**Comment by diizzyy** [comment]
git:// isn't really encouraged for usage due to issues with corp firewalls and/or proxies hence why it's very rarely used in the distro/repo overall.

**Comment by neheb** [comment]
switched to HTTP. HTTPS could also be used as the build system currently uses curl's --insecure parameter but that could change.

**Comment by cotequeiroz** [comment]
I just want to add that alpine 2.21 adds openssl 1.1 compatibility, since I'm working on making packages compatible with it.

**Comment by cotequeiroz** [comment]
@hnyman, you've been helping me with many packages.  Can you please take a look at this?

**Pull request merged**